tests/ImageSharp.Benchmarks/Codecs/Jpeg/DecodeJpeg_ImageSpecific.cs

```diff
@@ -47,12 +47,15 @@ public ShortClr()
         [Params(
             TestImages.Jpeg.BenchmarkSuite.Lake_Small444YCbCr,
             TestImages.Jpeg.BenchmarkSuite.BadRstProgressive518_Large444YCbCr,
-            /* The scaled result for the large image "ExifGetString750Transform_Huge420YCbCr"
-             is almost the same as the result for Jpeg420Exif,
-             which proves that the execution time for the most common YCbCr 420 path scales linearly.
-             TestImages.Jpeg.BenchmarkSuite.ExifGetString750Transform_Huge420YCbCr,
-             */
+
+#pragma warning disable SA1115 // Parameter should follow comma
```

Comment:
Should we not just disable these kind of rules instead?

Reply:
SA1115  is now disabled